js/index.js

```diff
@@ -44,26 +44,97 @@ function renderPepperoni() {
 
 function renderMushrooms() {
   // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(oneMush => {
+    if (state.mushrooms) {
+      oneMush.style.visibility = 'visible';
+    } else {
+      oneMush.style.visibility = 'hidden';
+    }
+  });
 }
 
 function renderGreenPeppers() {
   // Iteration 1: set the visibility of `<section class="green-pepper">`
+  document.querySelectorAll('.green-pepper').forEach(oneGP => {
+    if (state.greenPeppers) {
+      oneGP.style.visibility = 'visible';
+    } else {
+      oneGP.style.visibility = 'hidden';
+    }
+  });
 }
 
+
 function renderWhiteSauce() {
   // Iteration 2: add/remove the class "sauce-white" of `<section class="sauce">`
+  document.querySelectorAll('.sauce').forEach(oneSauce => {
+    if (state.whiteSauce) {
+      oneSauce.classList.add('sauce-white');
+    } else {
+      oneSauce.classList.remove('sauce-white');
+    }
+  });
 }
 
+// let changeCrust = document.getElementsByClassName("crust");
+// changeCrust[0].classList.remove("crust-gluten-free");
 function renderGlutenFreeCrust() {
   // Iteration 2: add/remove the class "crust-gluten-free" of `<section class="crust">`
+  document.querySelectorAll('.crust').forEach(oneCrust => {
+    if (state.glutenFreeCrust) {
+      oneCrust.classList.add("crust-gluten-free")
+    } else {
+      oneCrust.classList.remove("crust-gluten-free")
+    }
+  });
 }
```

Comment:
On this both functions you could have used `querySelector` because there is just one element `.sauce` and one `.crust`.

`
function renderWhiteSauce() {
  const oneSauce = document.querySelector(".sauce");
  if (state.whiteSauce) {
    oneSauce.classList.add("sauce-white");
  } else {
    oneSauce.classList.remove("sauce-white");
  }
}
`

`
function renderGlutenFreeCrust() {
  const oneCrust = document.querySelector(".crust");
  if (state.glutenFreeCrust) {
    oneCrust.classList.add("crust-gluten-free");
  } else {
    oneCrust.classList.remove("crust-gluten-free");
  }
}
`